VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -650,7 +650,7 @@ def do_status_report(self, operation, status, status_code, message, taskId = Non
         stat_rept.timestampUTC = date_place_holder
         date_string = r'\/Date(' + str((int)(time_span)) + r')\/'
         stat_rept = "[" + json.dumps(stat_rept, cls = ComplexEncoder) + "]"
-        stat_rept = stat_rept.replace('\\\/', '\/')  # To fix the datetime format of CreationTime to be consumed by C# DateTimeOffset
+        stat_rept = stat_rept.replace(r'\\\/', r'\/') # To fix the datetime format of CreationTime to be consumed by C# DateTimeOffset
```

Comment:
same here. this kind of change is repeated. Please add to PR description why this additional change is require apart from importlib failure handling.